test/test_unistd.rs

```diff
@@ -112,8 +112,10 @@ fn test_getpid() {
 fn test_getsid() {
     let none_sid: ::libc::pid_t = getsid(None).unwrap().into();
     let pid_sid: ::libc::pid_t = getsid(Some(getpid())).unwrap().into();
+    let plain_sid: ::libc::pid_t = getsid(getpid()).unwrap().into();
```

Comment:
None of these test cases cover `0`. I'm a fan of removing that functionality, but wanted to point it out for completeness.